examples/official-site/sitemap.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://sql.datapage.app/</loc>
+    <lastmod>2024-10-10</lastmod>
+    <priority>1.00</priority>
+  </url>
+  <url>
+    <loc>https://sql.datapage.app/component.sql</loc>
+    <lastmod>2024-10-10</lastmod>
+    <priority>0.80</priority>
+  </url>
+  <url>
+    <loc>https://sql.datapage.app/documentation.sql</loc>
+    <lastmod>2024-10-10</lastmod>
+    <priority>0.80</priority>
+  </url>
+  <url>
+    <loc>https://sql.datapage.app/your-first-sql-website/</loc>
+    <lastmod>2024-10-10</lastmod>
+    <priority>0.80</priority>
+  </url>
```

Comment:
Is it ok to have such a little subset of the site in the sitemap ? Won't it penalize other pages ?

We should probably either not have a sitemap, or have one that is generated dynamically in sql